Documentation changes in Normal Gamma and JointRV

mijo2 opened the pull request:
#### References to other Issues or PRs

Closes #15063

#### Brief description of what is fixed or changed

--> Updated the documentation in two functions, NormalGamma and JointRV in stats/joint_rv_types.py

#### Other comments


#### Release Notes

<!-- BEGIN RELEASE NOTES -->
* stats
  * Updated some documentation in NormalGamma and JointRV functions
<!-- END RELEASE NOTES -->

mijo2 committed:
stats: Documentation changes in joint_rv_types.py

asmeurer commented on sympy/stats/joint_rv_types.py:
```diff
@@ -251,12 +254,25 @@ def NormalGamma(syms, mu, lamda, alpha, beta):
     ==========
 
-    syms: list/tuple/set of two symbols for identifying each component
+    syms: A symbol/str for identifying the random variable.
```
Was this just wrong before? If it's really only supposed to be one symbol the parameter should be called `sym`.

mijo2 replied:
I think it was wrong before as when going into function definitions of ```multivariate_rv```, it is clearly mentioned that it takes only **a** symbol, not multiple ones. I will update ```syms``` to ```sym```.

mijo2 committed:
stats: Updated docstring of NormalGamma function

NormalGamma function's docstring and first parameter was syms even
though the helper function that it calls only asks for a singular
symbol. Thus, the syms was changed to sym in NormalGamma function
of joint_rv_types.py

oscarbenjamin commented:
If this is based on #15063 then it is important to keep the author attribution correct. For that you would need to start your commits from the other PR.

mijo2 commented:
@oscarbenjamin so I should push my commits to #15063 right?

oscarbenjamin commented:
> @oscarbenjamin so I should push my commits to #15063 right?

The normal way to revive an old PR is to rebase it to current master and then add your own commits:
```
# Make your own branch:
git remote add othername https://github.com/othername
git fetch
git checkout othername/otherbranch
git checkout -b mybranch
# Assuming that upstream is sympy/sympy.git
git fetch upstream
git rebase upstream/master
#### Make your own changes, run tests, commit then
git push -u origin mybranch
#### Open a pull request
```

mijo2 commented:
@oscarbenjamin I revived the PR as you asked in a new PR. #18589
